packages/wallet-sdk/src/util/provider.ts

```diff
@@ -12,7 +12,11 @@ export async function fetchRPCRequest(request: RequestArguments, rpcUrl: string)
     method: 'POST',
     body: JSON.stringify(requestBody),
     mode: 'cors',
-    headers: { 'Content-Type': 'application/json', 'X-Cbw-Sdk-Version': LIB_VERSION },
+    headers: {
+      'Content-Type': 'application/json',
+      'X-Cbw-Sdk-Version': LIB_VERSION,
+      'X-Cbw-Sdk-Platform': LIB_NAME,
```

Comment:
should this be `X-Cbw-Sdk-Name` for consistent naming?

Reply:
Also okay renaming the LIB_NAME var

Reply:
this header param is for BE to distinguish web/mobile SDKs, updating this name may need BE and mobile sdk update, let's keep it as it is?

Reply:
sounds good